### DISCUSSION TITLE
Rename GitHub Milestone(s) to proper version

### DISCUSSION THREAD

**Discussion by SebastianOpriel** [eclipse-edc/Connector#3227] [Ideas]
As I understood, Milestone releases were switched to sem versioning. Maybe we can rename the GitHub Milestone (https://github.com/eclipse-edc/Connector/milestone/12) or delete it. This led to some discussions and confusions 🙄

**Comment by ndr-brt** [comment]
The "milestone" represent an iteration cycle (6-8 weeks) with its own planning, before we released at the end of the milestone, now the release is independent from the milestone cycle, thus why we dropped the "milestone" word from the release.

**Reply by SebastianOpriel** [reply]
Okay, good approach.
Here is still a milestone called milestone, which might be renamed/closed as well: https://github.com/eclipse-edc/Connector/milestone/13

**Reply by ndr-brt** [reply]
nope, we'll keep the milestone concept, it makes the community aware of what's currently on development and what is the expected delivery date.

**Reply by SebastianOpriel** [reply]
Maybe we talk about cross purposes. Are you talking about naming milestones or the GitHub Feature of milestones? I totally agree to use GitHub Milestones feature like [Kubernetes](https://github.com/kubernetes/kubernetes/milestones) does. I was only referring to the naming convetions of EDC's GitHub Milestones, which might be changed to something like "v0.3.0":
![image](https://github.com/eclipse-edc/Connector/assets/22075788/65e2d14f-899b-459b-b599-00aac7a8aa70)

**Reply by juliapampus** [reply]
We are talking about the naming. A milestone lasts some weeks and matches our current work planning, as our releases are request- and resource-based and not pre-scheduled (this can ofc change in the future). This way, one milestone usually covers multiple 0.0.X releases. Therefore, we provide information about what 0.X.0 release will cover all milestone changes (this was the decision we made when we discussed your request):
![image](https://github.com/eclipse-edc/Connector/assets/72392527/61a10777-f150-450e-a355-21fc3e9cf144)
For all 0.0.X releases, we ask our users to check out the corresponding release notes in order to know what changes are already covered or not.

**Reply by juliapampus** [reply]
Including the version in the milestone description allows us to change the specific version any time. This is why we do not name the GH milestone accordingly. One step at a time ...

**Comment by mspiekermann** [comment]
Closed as @ndr-brt and @juliapampus laid out our approach to naming milestones and releases.
